### PR TITLE
moving options to attributes and updating demo

### PR DESCRIPTION
**Pull request by rosemariesadler**
* change `fullscreen-on-mobile=false` to `no-fullscreen-mobile`
* move options to attributes
* add slightly more docs to the top level demo
* add `no-close-button` feature
* update demo

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -146,11 +146,14 @@
                 <select name="transition" id="transition">
                   <option value=""></option>
                   <option value="from-bottom">from-bottom</option>
-                  <option value="from-left">from-left</option>
+                  <option value="from-right">from-right</option>
```
Comment (skye2k2) [inline demo/index.html]: Are there no options to slide in from top or left?

**Comment by skye2k2** [comment]
I've got empty attributes and form reset now working.

**Comment by skye2k2** [comment]
So, the focusback functionality changed for the demo page--if I use enter to open a dialog, when closing, focus is always given back to the `Open Dialog` button.

File: demo/index.html (hunk 1 above)
Comment (rosemariesadler) [reply demo/index.html]: No - but they wouldn't be hard to add - I just don't think there's a use case for them, but as soon as there is, we can add it.

**Comment by rosemariesadler** [comment]
That's because there was a spelling error with `focusBackElement` that Jason fixed, so now its working how we thought we were telling it to. We can change it so that we intentionally use the active element.

**Pull request merged**